nixops/backends/__init__.py

```diff
@@ -329,6 +339,20 @@ def send_keys(self) -> None:
             )
             self.run_command("mv " + tmp_outfile_esc + " " + outfile_esc)
             os.remove(tmp)
+
+        def make_task(key: str, opts: Dict[str, Any]) -> nixops.parallel.Task:
+            task = types.SimpleNamespace()
+            task.name = self.name
+            task.key = key
+            task.opts = opts
+            return task
+
+        nixops.parallel.run_tasks(
+            # by default ssh allows 10 sessions per connection via MaxSessions setting
+            nr_workers=8,
```

Comment:
Some day, somebody will send a PR making the 8 configurable for some weird edge case. Today is not that day :P

Reply:
:D